### ISSUE TITLE
FormBuilderChipsInput Error using required validator

### ISSUE THREAD

**Issue by socramm** [danvick/flutter_form_builder#611]
It works fine if I don't use `validator: FormBuilderValidators.compose([
                    FormBuilderValidators.required(context),
                  ]),`

<img width="1919" alt="works" src="https://user-images.githubusercontent.com/75392108/100923994-50ed9c00-34e0-11eb-9c2b-138c852a4352.png">

But as soon als I do I get this Error Message

<img width="1920" alt="error" src="https://user-images.githubusercontent.com/75392108/100923989-4f23d880-34e0-11eb-89e1-11b848cd4b84.png">

`flutter_form_builder: ^4.0.2`

**Comment by byzramku** [comment]
Do u really use ^4.0.2 ?

**Comment by socramm** [comment]
Yes my Version is até the bottom of my post. 

`flutter_form_builder: ^4.0.2`

**Comment by Artur110** [comment]
It's not only in chips input. Same issue with text field input.

**Comment by awhitford** [comment]
BTW...  You don't need to use `compose` when you only have 1 validator.

**Comment by awhitford** [comment]
Is it possible that your problem is line 65:
```dart
return [];
```
Maybe that needs a type, like:
```dart
return <String>[];
```
And actually, adding `const` might be smart too:
```dart
return const <String>[];
```

**Comment by awhitford** [comment]
Related to my last comment, I noticed that there was a missing type declaration here:
https://github.com/danvick/flutter_form_builder/blob/c2b63433778187cd9258499f1bf5f53781fa3ffa/lib/src/fields/form_builder_chips_input.dart#L9

It should say:
```dart
 final ChipsInputSuggestions<T> findSuggestions; 
```

I submitted PR #621 to address that.  That should add the necessary type constraint, making my earlier suggestion irrelevant.  However, in the meantime (until the PR is merged into a Release), adding the type declaration should do the trick.
